nss-day-cohort-43/nuttyshell-sons-of-witches
# Bs working deletion functionality

# Description
Add deletion function to the Chat box so Chat can now properly fetch, post, and delete.

Fixes # (issue)
## Fixed the functionality for the Delete button so messages may now be deleted only by the user who submitted them.

Please delete options that are not relevant.

# Testing Instructions
Please test this function by clicking adding a message, saving the message by clicking the Submit Message button, ensuring the message you entered appears along with a Delete button, and that you can delete that message by clicking the Delete button.

# Checklist:
